src/report_workflow/report_args.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import argparse
+import logging
+
+from test_workflow.test_kwargs import TestKwargs
+
+
+class ReportArgs:
+    test_run_id: str
+    keep: bool
+    test_manifest_path: str
+    artifact_paths: dict
+    test_type: str
+    logging_level: int
+
+    def __init__(self) -> None:
+        parser = argparse.ArgumentParser(description="Report for test on OpenSearch Bundle")
+        parser.add_argument("test_manifest_path", type=str, help="Specify a test manifest path.")
+        parser.add_argument("-p", "--artifact-paths", nargs='*', action=TestKwargs, default={},
```

Comment:
Is this the path to the test data?

Reply:
This is the path to the artifacts that were used for the test workflow. This argument is used to retrieve the information of distribution manifest so that we would know which distribution bundle the test workflow was running on.

Reply:
Why not take only the distribution manifest path then instead of entire artifacts path?

Reply:
We are triggering this workflow after test workflow. For the test workflow, we are using this same `artifacts_path` as argument; so keeping the same argument content for this following workflow would be easy and consistent IMO.

Reply:
Both are completely different things. We do not need to be consistent. One is running the test other is generating the report for the test run. If all we need is manifest info I believe just take that instead of entire path. What else do we here from those paths?

Reply:
Agree with Sayali, if the only aim is to get the components that were tested then just use the dist manifest path.

Reply:
Yeah make sense. I will update it here since this workflow should be considered as an individual workflow. Thanks.

Reply:
Just had a discussion with @gaiksaya, this artifact paths would also be used for the command in the test-run manifest which show the shell commands to reproduce testing. That would include `artifacts path` as in the test workflow. We would want to use this argument for that as well.